Makefile

```diff
@@ -14,6 +14,8 @@ else
 endif
 PROTOC_URL := https://github.com/protocolbuffers/protobuf/releases/download/v28.0/$(PROTOC_ZIP)
 
+PROTOC_BIN := /usr/local/bin/protoc
```

Comment:
Isn't this macOS specific?

Reply:
this is the path where we put the bin (see line 50). Btw i'll also replace the path there with this variable

Reply:
True. The other option is to provide it through environment variable or default to PROTOC_BIN.

For linux it could be something like:
```
make generate-protobuf PROTOC_BIN=/usr/bin/protoc
```